unison-cli/integration-tests/IntegrationTests/print.u

```diff
@@ -1,2 +1 @@
-print : '{IO, Exception} ()
-print _ = base.io.printLine "ok"
```

Comment:
There's no base available here,  but we're not testing base, just that we can parse a transcript.

Reply:
I'm a little late to the party (this PR was merged a few weeks back), but I think if we have this scratch file called `print`, but we take out the printing, then we should do some combination of:
* rename the file
* try to figure out if this is being used somewhere else to confirm that printing works
* if we do need to print, we can do so without base, by copying that definition into the file